website/static/js/profile.js

```diff
@@ -924,6 +929,11 @@ ListViewModel.prototype.unserialize = function(data) {
         self.editable(false);
     }
     self.contents(ko.utils.arrayMap(data.contents || [], function (each) {
+        for (var attr in each) {
```

Comment:
That's probably fine then, I mentioned to @fabmiz to look into that in this case since I know weird things can happen with for...in.